locals.pkr.hcl

```
@@ -0,0 +1,32 @@
+locals {
+  now_unix_timestamp    = formatdate("YYYYMMDDhhmmss", timestamp())
+  agent                 = format("%s-%s", var.agent_os_type, var.agent_os_version)
+  agent_os_version_safe = replace(var.agent_os_version, ".", "_")
+  image_name            = format("jenkins-agent-%s-%s", var.agent_os_type, var.agent_os_version)
+  aws_instance_type = {
+    "amd64" = "t3.large"  # 2 CPU / 1 GB / $0.0104
```

```suggestion
    "amd64" = "t3.large"  # 2 CPU / 8 GB / $0.0104
```
From https://aws.amazon.com/ec2/instance-types/t3/

Not sure about the price, didn't found any on https://aws.amazon.com/ec2/instance-types/t4/ to compare and select one.

---

https://aws.amazon.com/fr/ec2/instance-types/t3/#Product_Details

```suggestion
    "amd64" = "t3.large"  # 2 CPU / 8 GB / $0.0835 - https://aws.amazon.com/fr/ec2/instance-types/t3/#Product_Details
```